### DISCUSSION TITLE
Require a detailed log for submission Error

### DISCUSSION THREAD

**Discussion by dongyc98** [PaddlePaddle/PaddleSpatial#257] [BaiduKDDCup2022] [answered]
Hi, My team name is GoGoTeam, 
I run the evaluation.py in my local server and it runs smoothly and give a final score correctly, but when I submit to the online test, it throws Exception '异常' without detailed error message. I have checked code for several times. Please help us provide detailed error log.
![image](https://user-images.githubusercontent.com/36831594/178635467-29dda358-e1da-4b57-b24e-4febec81545e.png)

**Comment by ramber1836** [comment, marked as answer]
usage: eval_main.py [-h] [--checkpoints CHECKPOINTS] [--start_col START_COL]
                    [--pred_file PRED_FILE] [--framework FRAMEWORK]
                    [--graph_file GRAPH_FILE]
eval_main.py: error: unrecognized arguments: --submit_file /home/aistudio/dgl.zip

**Reply by ramber1836** [reply]
No argument parser is allowed in your code. Please read the Submission Manual carefully.
